### ISSUE TITLE
[CM13][smultron] bluetooth pairing not possible

### ISSUE THREAD

**Issue by alexander-n8hgeg5e** [LegacyXperia/local_manifests#1045]
If I try to pair with my pc or my car radio and the method with 6 numbers pin showing on both devices  is used:
The Phone shows the message the pin is wrong and doesn't  store the pairing.
While the other devices (car radio )stores the pairing. With my pc the pc deletes the pairing after some seconds.

**Comment by realcatgirly** [comment]
Yep, BT is buggy as hell. I need to pair my bluetooth ear buds every day I use them
